index.js

```diff
@@ -38,8 +38,22 @@ alfy.alfred = {
 
 alfy.input = process.argv[2];
 
-alfy.output = arr => {
-	console.log(JSON.stringify({items: arr}, null, '\t'));
+const isDefined = x => x !== null && x !== undefined;
+
+const formatItemArg = item => {
+	if (isDefined(item) && isDefined(item.variables)) {
+		const alfredworkflow = {arg: item.arg, variables: item.variables};
+		item.arg = JSON.stringify({alfredworkflow});
+		delete item.variables;
+	}
+};
+
+alfy.output = items => {
+	if (isDefined(items) && items.forEach) {
```

Comment:
This would be a breaking change to the `output` API, right? 

Old implementation:
```js
alfy.output('hi!'); // no problem
```

New implementation:
```js
alfy.output('hi!'); // TypeError
```

I'm happy to implement it, just want to confirm. 

P.S. - does Alfy follow SemVer?